packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -386,7 +391,9 @@ describe("editable-tree", () => {
         };
         const forest = setupForest(schemaData, [{ type: optionalChildSchema.name, fields: { child: [{ type: int32Schema.name, value: undefined }] } }]);
         const [context, field] = getEditableTree(forest);
-        assert.throws(() => ((field as EditableTree).child), /`undefined` values not allowed for primitive fields/);
+        assert.throws(() => ((field as EditableTree).child),
```

Comment:
this shouldn't be an assert i don't think. we should file a bug against tree to not use asserts for user facing errors

Reply:
this is specifically detecting that if out of schema data gets into our system somehow, that this case errors (using undefined where a number is required). As we intend to have public APIs that ensure data is in schema, this catches violations of internal invariants before they could manifest as broken data exposed to the user.

An argument could be made for this being a document corruption scenario, but since at least currently it's possible to use these APIs standalone (without hooking up a real document), that seems a bit odd.